### ISSUE TITLE
Import Error: Workbook is not a constructor

### ISSUE THREAD

**Issue by 99899899998** [xmindltd/xmind-sdk-js#42]
hey, i want to know hot to use it in my project
I follow the steps:
npm i xmind
import { Workbook, Topic, Marker, Zipper } from 'xmind';
but it reported an error " xmind__WEBPACK_IMPORTED_MODULE_0__.Workbook is not a constructor "

Hope to get a reply thanks

**Comment by danielsss** [comment]
Need more information to confirm what was happened.

1. Environment
2. JS or Typescript
3. Make sure the SDK is correctly installed.

**Comment by danielsss** [comment]
No more further

**Comment by kiroog** [comment]
@danielsss Hey，I am confronted with the same issue in my vue/ts project build via webpack. I'm sure I have correctly installed the SDK for I have tried several project. It seems that the import content is empty.

**Comment by danielsss** [comment]
Hi @kiroog, 

Could you provide more details about this issue? or could you stick the object details of what you imported to here?

**Comment by kiroog** [comment]
> Hi @kiroog,
> 
> Could you provide more details about this issue? or could you stick the object details of what you imported to here?

hi @danielsss , sorry for leaving this for couple days
my steps are as follow in my vue/ts project,
```
npm i xmind
import { Workbook, Topic, Marker, Zipper } from 'xmind';
```
and after confronting with the error " xmind__WEBPACK_IMPORTED_MODULE_0__.Workbook is not a constructor " I tried to print the items I import in the console but got `undefined` for all of them

**Comment by danielsss** [comment]
@kiroog I've released v2.2.27(#53) with several bug fixes for your inquiry and it's tested on Vue.js 3.

Please, let me know if you still have other problems.

**Comment by kiroog** [comment]
> @kiroog I've released v2.2.27(#53) with several bug fixes for your inquiry and it's tested on Vue.js 3.
> 
> Please, let me know if you still have other problems.
@danielsss it solved my problem! thanks for your incredible job!
